mkdocs/docs/user_guide.md

```diff
@@ -2699,7 +2701,7 @@ Compare Classifier Predictions
 
 ### compare_classifiers_predictions
 
-This visualization uses the `ground_truth`, `ground_truth_metadata`, `field`, `predictions` and `model_names` parameters.
+This visualization uses the `ground_truth`, `ground_truth`, `field`, `predictions` and `model_names` parameters.
```

Comment:
double check if this is correct, ground truth is repeted twice

Reply:
I have checked it is correct - The new cli function prototype does not need the ground truth metadata. You can also check the old compare_classifiers_predictions function declaration to see that it is not using that argument.
def compare_classifiers_predictions_cli(
        predictions,
        ground_truth,
        ground_truth_split,
        field,
        **kwargs
):

Reply:
I have removed the repeated word.